.vscode/settings.json

```diff
@@ -1,36 +0,0 @@
-{
-  "editor.fontSize": 12,
-  "editor.tabSize": 2,
-  "files.trimTrailingWhitespace": true,
-  "editor.wordWrap": "on",
-  "editor.formatOnPaste": true,
-  "html.format.enable": true,
-  "html.format.indentHandlebars": true,
-  "html.format.maxPreserveNewLines": 0,
-  // Set the default
-  "editor.formatOnSave": false,
-  // Enable per-language
-  "[javascript]": {
-    "editor.suggest.insertMode": "replace",
-    "editor.formatOnSave": true
-  },
-  "[css]": {
-    "editor.suggest.insertMode": "replace",
```

Comment:
@neilvarunnaidu Why did you delete this file ? This file is purposely set up to maintain proper formatting and styling.